spec.html

```diff
@@ -17492,6 +17495,15 @@
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-function-definitions-static-semantics-containsusestrict">
+      <h1>Static Semantics: ContainsUseStrict</h1>
+      <emu-see-also-para op="ContainsUseStrict"></emu-see-also-para>
```

Comment:
this is not a supported thing... trying to decide if I want to support this or if it's fine to say "See also <emu-xref aoid="ContainsUseStrict"></emu-xref>".